helium/src/scss/_forms.scss

```diff
@@ -1,20 +1,42 @@
 @import 'compass';
 @import "constants";
-@import "ycthemelib/responsive";
 
 $USER_FORMS_FIELD_HEIGHT: 37px;
-$FORM_BACKGROUND_COLOUR: #3c425c;
+//$FORM_BACKGROUND_COLOUR: #3c425c;
+$FORM_BACKGROUND_COLOUR: #FFF;
 $VERTICAL_MARGIN_BETWEEN_FIELDS: 52px;
 $FIELD_MAX_WIDTH: 800px;
 
+
+// Chrome/Opera/Safari
+::-webkit-input-placeholder {
+  color: #969696;
+  font-weight: bold;
+}
+
+// Firefox 19+
+::-moz-placeholder {
+  color: #969696;
+  font-weight: bold;
+}
+
+// IE 10+ ??
+:-ms-input-placeholder {
+  color: #969696 !important;
+  font-weight: bold;
+}
+
+// Firefox 18-
+:-moz-placeholder {
+  color: #969696;
+  font-weight: bold;
+}
+
```

Comment:
Why repeat all the styling for the identical ones (simply have a comma separated list of selectors and one common styling bloc.